static/js/components/LearnerSearch.js

```diff
@@ -128,6 +129,16 @@ export default class LearnerSearch extends SearchkitComponent {
       {openLearnerEmailComposer: this.props.openLearnerEmailComposer},
       LearnerResult
     );
+    this.forceSearch = true;
+  }
+
+  loaded = (): boolean => {
+    if (this.forceSearch) {
```

Comment:
because the component below initiate the search it self, if we dont render them search query wont call